applications/asset_tracker_v2/src/ext_sensors/ext_sensors.c

```diff
@@ -20,20 +20,34 @@
 LOG_MODULE_REGISTER(ext_sensors, CONFIG_EXTERNAL_SENSORS_LOG_LEVEL);
 
 /* Convert to s/m2 depending on the maximum measured range used for adxl362. */
-#if defined(CONFIG_ADXL362_ACCEL_RANGE_2G)
+#if IS_ENABLED(CONFIG_ADXL362_ACCEL_RANGE_2G)
 #define ADXL362_RANGE_MAX_M_S2 19.6133
-#elif defined(CONFIG_ADXL362_ACCEL_RANGE_4G)
+#elif IS_ENABLED(CONFIG_ADXL362_ACCEL_RANGE_4G)
 #define ADXL362_RANGE_MAX_M_S2 39.2266
-#elif defined(CONFIG_ADXL362_ACCEL_RANGE_8G)
+#elif IS_ENABLED(CONFIG_ADXL362_ACCEL_RANGE_8G)
 #define ADXL362_RANGE_MAX_M_S2 78.4532
 #endif
 
-#define ADXL362_THRESHOLD_RESOLUTION_DECIMAL_MAX 2048
-
-/* Local accelerometer threshold value. Used to filter out unwanted values in
- * the callback from the accelerometer.
+/** The datasheet suggests that the maximum is at 2000, not 2048.
+ *  This is derived from the sensitivity values.
  */
-double threshold = ADXL362_RANGE_MAX_M_S2;
+#define ADXL362_THRESHOLD_RESOLUTION_DECIMAL_MAX 2000
+
+#if   IS_ENABLED(CONFIG_ADXL362_ACCEL_ODR_12_5)
+#define ADXL362_TIMEOUT_MAX_S 5242.88
+#elif IS_ENABLED(CONFIG_ADXL362_ACCEL_ODR_25)
+#define ADXL362_TIMEOUT_MAX_S 2621.44
+#elif IS_ENABLED(CONFIG_ADXL362_ACCEL_ODR_50)
+#define ADXL362_TIMEOUT_MAX_S 1310.72
+#elif IS_ENABLED(CONFIG_ADXL362_ACCEL_ODR_100)
+#define ADXL362_TIMEOUT_MAX_S 655.36
+#elif IS_ENABLED(CONFIG_ADXL362_ACCEL_ODR_200)
+#define ADXL362_TIMEOUT_MAX_S 327.68
+#elif IS_ENABLED(CONFIG_ADXL362_ACCEL_ODR_400)
+#define ADXL362_TIMEOUT_MAX_S 163.84
+#endif
+
+#define ADXL362_TIMEOUT_RESOLUTION_DECIMAL_MAX 65536
```

Comment:
```suggestion
#define ADXL362_TIMEOUT_RESOLUTION_MAX 65536
```
Not sure we need the word DECIMAL there.